Tasks/Xcode/xcode.js

```diff
@@ -0,0 +1,198 @@
+/// <reference path="../../definitions/node.d.ts" />
+/// <reference path="../../definitions/Q.d.ts" />
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
+/// <reference path="../../definitions/ios-signing-common.d.ts" />
```

Comment:
We shouldn't check in the js.  It's produced by the build.

As a matter of fact, I think we ignore all js files in .gitignore except the one from here so you should edit .gitignore as well.